Automattic/wp-calypso
# Gutenframe: start Gutenlypso removal

#### Changes proposed in this Pull Request

After enabling Gutenframe in production all of the Gutenlypso code
is now unreachable. In this patch we are starting the removal of the
majority of that code. The work on cleaning up the styles and remaining
packages will be handled in follow up PRs.

#### Testing instructions

1. Navigate to http://calypso.localhost:3000/block-editor/post/.
2. Make sure that Placeholder styles are loading properly.
3. Try publishing a new post.
4. Make sure that Media Library, Revisions and other iframe integrations work.
5. Try editing an existing post.
6. Test opt-in and opt-out flows.
7. Check for visual regressions.
8. Verify that Custom Post Types work as expected.
